### PR TITLE
Preceding invoice Infos

### PR DESCRIPTION
**Pull request by TomcatBB**
For Invoice Corrections it is mentioned that the Preceding Invoice Infos like the Invoice No and the Issue Date should be filled. 

Change adds fields for Preceding Invoice CII Group: InvoiceReferencedDocument

### CODE REVIEW

File: ZUGFeRD/InvoiceReferencedDocument.cs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+﻿using System;
```
Comment (stephanstapel) [inline ZUGFeRD/InvoiceReferencedDocument.cs]: could you add the license on top of the file?

File: ZUGFeRD/InvoiceDescriptor21Writer.cs [hunk 2]
```diff
@@ -572,235 +572,251 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream)
             #endregion
 
             #region ApplicableHeaderTradeSettlement
-            Writer.WriteStartElement("ram:ApplicableHeaderTradeSettlement");
```
Comment (stephanstapel) [inline ZUGFeRD/InvoiceDescriptor21Writer.cs:573]: There are 364 changes to the InvoiceDescriptor21Writer.cs. Why is that?

File: ZUGFeRD/InvoiceReferencedDocument.cs (hunk 1 above)
Comment (TomcatBB) [reply ZUGFeRD/InvoiceReferencedDocument.cs]: I already added the license infos in a second commit, but the check still gives the error.
Do you know what else could cause the problem?

**Review by stephanstapel — APPROVED** [review]

File: ZUGFeRD/InvoiceDescriptor21Writer.cs (hunk 2 above)
Comment (TomcatBB) [reply ZUGFeRD/InvoiceDescriptor21Writer.cs:573]: I changed the indentation of the ApplicableHeaderTradeSettlement block for better reading. And this is seen as a change in each line. But everything I have done is added a tab for each line in the block/region

**Pull request merged**